src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,95 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String STRING_SEPARATOR = ",";
+    private static final String[] REPORT_COLUMNS = new String[] {"supply", "buy", "result"};
+    private static BufferedWriter bufferedWriter = null;
+    private static File file = null;
```

Comment:
Using static fields `bufferedWriter` and `file` is not recommended here. These should be instance variables to avoid potential concurrency issues when multiple instances of `WorkWithFile` are used.